README.md

```diff
@@ -292,7 +292,7 @@ The MacBookPro13,1 comes with a `Broadcom Limited BCM4350 802.11ac Wireless
 Network Adapter` which works fine out of the box using the `brcmfmac` driver
 (ensure you got the matching firmware package installed).
 
-The MacBookPro13,2 and MacBookPro13,3 come with a `Broadcom Limited BCM43602
+The MacBookPro13,2, MacBookPro13,3, MacBookPro14,2, and MacBookPro14,4 come with a `Broadcom Limited BCM43602
```

Comment:
Typo? `MacBookPro14,4` should probably be `14,3`

Reply:
Whoops, you're right! Surprised I didn't catch that given how small the commit is <.<

Reply:
See #43 for a fix